helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -1215,6 +1216,7 @@ operator:
       python:
```

Comment:
Added a note about Python and ApacheHttpD instrumentation above

Reply:
done